### PR TITLE
Upgrade to Webpack 5

### PR DESCRIPTION
**Pull request by himanshu007-creator**
#151

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -4,6 +4,11 @@ module.exports = {
       'i.scdn.co'
     ]
   },
+  module.exports = {
```
Comment (doamatto) [inline next.config.js:7]: This was already declared in the first line of the document

File: next.config.js [hunk 2]
```diff
@@ -4,6 +4,11 @@ module.exports = {
...
+  future: {
+    webpack5: true,
+  },
+}
   webpack: (config) => {
     require('./lib/sitemap.js')
     config.node = {
```
Comment (doamatto) [inline next.config.js:14]: This section would need to be altered to work with Webpack 5. Iirc it has to do with [`resolve.fallback`](https://webpack.js.org/configuration/resolve/#resolvefallback), to help point you in the right direction

**Review by doamatto — CHANGES_REQUESTED** [review]

**Comment by himanshu007-creator** [comment]
@doamatto  , can you tell the exact code that needs to be put  please?

**Comment by doamatto** [comment]
Sorry Himanshu, I'm not exactly sure what needs to be put; hence why I made the issue for me or someone else to get to later. It'll take a little bit of researching in Webpack's documentation to find out how to get it to work properly.

**Comment by doamatto** [comment]
Resolved with #157.

**Pull request closed without merging**